common/src/main/cpp/extensions/openxr_fb_composition_layer_settings_extension_wrapper.cpp

```diff
@@ -100,6 +105,13 @@ uint64_t OpenXRFbCompositionLayerSettingsExtensionWrapper::_set_viewport_composi
 
 	settings->layerFlags = 0;
 
+	if (meta_automatic_layer_filter && p_property_values.get(ENABLE_AUTO_FILTER_PROPERTY_NAME, false) && (int)p_property_values.get(AUTO_OPTIONS_PROPERTY_NAME, 0)) {
+		settings->layerFlags |= XR_COMPOSITION_LAYER_SETTINGS_AUTO_LAYER_FILTER_BIT_META;
+		settings->layerFlags |= (int)p_property_values.get(AUTO_OPTIONS_PROPERTY_NAME, 0);
+
+		return reinterpret_cast<uint64_t>(settings);
```

Comment:
I imagine we return here because the auto options override the manual ones; can you add a comment describing that behavior.

Reply:
Added a comment above the if statement in latest push!